Upstream: no
Source: olesalscheider
Reason: make install of all qmake based projects fails otherwise under sydbox

--- a/src/corelib/io/qtemporaryfile_p.h	2018-07-29 14:24:21.249582046 +0200
+++ b/src/corelib/io/qtemporaryfile_p.h	2018-07-29 14:25:04.511065252 +0200
@@ -58,17 +58,6 @@
 #include "private/qfile_p.h"
 #include "qtemporaryfile.h"
 
-#if defined(Q_OS_LINUX) && QT_CONFIG(linkat)
-#  include <fcntl.h>
-#  ifdef O_TMPFILE
-// some early libc support had the wrong values for O_TMPFILE
-// (see https://bugzilla.gnome.org/show_bug.cgi?id=769453#c18)
-#    if (O_TMPFILE & O_DIRECTORY) == O_DIRECTORY
-#      define LINUX_UNNAMED_TMPFILE
-#    endif
-#  endif
-#endif
-
 QT_BEGIN_NAMESPACE
 
 struct QTemporaryFileName
